plugins/lookup/tss.py

```diff
@@ -157,10 +169,35 @@
   tasks:
       - ansible.builtin.debug:
           msg: the password is {{ secret_password }}
+
+#Private key stores into certificate file which is attached with secret.
+#If fetch_attachments=True then private key file will be download on specified path
+#And file content will display in debug message.
```

Comment:
Nit picking (and surprised pylint has not nit-picked that before me), but code convention is to add a space after the `#`:
```suggestion
# Private key stores into certificate file which is attached with secret.
# If fetch_attachments=True then private key file will be download on specified path
# And file content will display in debug message.
```

Reply:
pylint doesn't care since this is part of a YAML comment - and ansible-test's yamllint config is very, very lenient.